### PR TITLE
pipeline fix

### PR DESCRIPTION
**Pull request by sczembor**
Closes #x

## Proposed Changes

- update workflows to additionally install `poetry` package
- update `Pipefile.lock`

## TODO

- [ ] update change log

### CODE REVIEW

File: .github/workflows/01-main.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ on:
 
 jobs:
   tests:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```
Comment (rmeena840) [inline .github/workflows/01-main.yaml:18]: Was this causing the issue?

**Review by rmeena840 — APPROVED** [review]
Approved

File: .github/workflows/01-main.yaml (hunk 1 above)
Comment (sczembor) [reply .github/workflows/01-main.yaml:18]: LMK what do you think and lets merge it

**Review by Megha-Dev-19 — APPROVED** [review]
Thanks for the fixes

**Pull request merged**